cmd/restic/global.go

```diff
@@ -571,6 +579,10 @@ func parseConfig(loc location.Location, opts options.Options) (interface{}, erro
 			cfg.ProjectID = os.Getenv("GOOGLE_PROJECT_ID")
 		}
 
+		if cfg.ProjectID == "" {
```

Comment:
Judging from the code the project id is only necessary to initialize a repository. Have there been complaints from people not passing the project id?